### ISSUE TITLE
Contract estimation integration tests hard codes fees 

### ISSUE THREAD

**Issue by arvidj** [ecadlabs/taquito#2168]
**Description**

The contract estimation tests `contract-estimation-tests.spec.ts` hard codes the estimated fees. I'm unsure how the estimation works but I suspect they depend on protocol parameters that are not the same on mainnet/testnets w.r.t. local sandboxes. This makes it hard to run this test against local sandboxed networks where fees might vary.

**Steps To Reproduce**

1. Start a local sandboxed octez node with RPC endpoint http://localhost:8732 running protocol alpha
2. Run the integration tests, e.g. `TEZOS_RPC_MONDAYNET=http://localhost:8732 RUN_MONDAYNET_WITH_SECRET_KEY=true npm run -w integration-tests test -- --reporters=default --reporters=jest-junit contract-estimation-tests.spec.ts`
3. It will fail.

**Expected behavior**

For example, the test might detect when run against "true" mondaynet/jakartanet and only run in that case.

**Additional context**

 This is an issue in TeCI.

**Comment by Innkst** [comment]
#2163 may fix this issue as well. Will need to check after.

**Comment by arvidj** [comment]
> #2163 may fix this issue as well. Will need to check after.

Yes, at least the changes in #2164 adds a `NetworkType` that we can use to disable these tests. Once/if #2164 is merged I'll propose an MR to that effect.
